packages/widgetbook/test/src/addons/test_scale_factor/text_scale_factor_test.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+import 'package:widgetbook/src/addons/text_scale_addon/text_scale_provider.dart';
+import 'package:widgetbook/src/addons/text_scale_addon/text_scale_selection_provider.dart';
+import 'package:widgetbook/src/workbench/renderer.dart';
+import 'package:widgetbook/widgetbook.dart';
+
+import '../utils/addons.dart';
+import '../utils/custom_app_theme.dart';
+import '../utils/theme_wrapper.dart';
+import 'text_scale_test_utilities.dart';
+
+void main() {
+  late Renderer sut;
+  final rendererKey = GlobalKey();
+  const mediaQueryKey = Key('MediaQuery');
+
+  setUp(() {
+    sut = Renderer(
+      key: rendererKey,
+      appBuilder: (context, child) {
+        return MediaQuery(
+          key: mediaQueryKey,
+          data: MediaQuery.of(context).copyWith(
+            textScaleFactor: context.textScale,
+          ),
+          child: child,
+        );
+      },
+      useCaseBuilder: (context) => Text(
+        'Text scale factor',
+        key: textKey,
+        textScaleFactor: context.textScale,
```

Comment:
Do we need this? I think the customer won't use it this way

Reply:
seems i cant get textscale factor from MediaQuery

I get a null value